addons/web/static/src/views/fields/dynamic_placeholder_hook.js

```diff
@@ -11,7 +11,7 @@ export function useDynamicPlaceholder(elementRef) {
     let closeCallback;
     let positionCallback;
     const popover = usePopover(DynamicPlaceholderPopover, {
-        onclose: () => closeCallback?.(),
+        onClose: () => closeCallback?.(),
```

Comment:
maybe we should merge it directly in master ?

Reply:
https://github.com/odoo/odoo/pull/171167